### DISCUSSION TITLE
Hide Tabs and Search Bar in Chrome

### DISCUSSION THREAD

**Discussion by sofia-bg** [UNC-Project-Open-AAC/OS-DPI#219] [General]
Finally took the time to figure out how to hide the tabs and search bar in chrome, and at least on a Mac when in full screen the command is: command-shift-f 

@aetkie

**Comment by aetkie** [comment]
Helpful! Thanks for sharing

**Comment by gbishop** [comment]
It is also in the "hamburger" menu at the end of your URL bar, next to Zoom. It is the box made of 4 corners on the right. 

![Screenshot from 2023-09-11 14-31-30](https://github.com/UNC-Project-Open-AAC/OS-DPI/assets/198012/00543cb9-8fb8-40ab-9479-b3f79eb326e5)
